pkg/controllerutils/miscelleneous.go

```diff
@@ -26,8 +26,9 @@ const separator = ","
 // AddTasks adds a task to the ShootTasks annotation of the passed map.
 func AddTasks(existingAnnotations map[string]string, tasksToAdd ...string) {
 	var tasks []string
-	if len(existingAnnotations[common.ShootTasks]) > 0 {
-		tasks = strings.Split(existingAnnotations[common.ShootTasks], separator)
+	existingTasks, _ := common.GetTasksAnnotation(existingAnnotations)
```

Comment:
BTW: typo in the filename.
Can you rename `pkg/controllerutils/miscelleneous.go` to `...miscellaneous.go`?